packages/context/src/binding.ts

```diff
@@ -145,16 +145,35 @@ export class Binding {
     };
   }
 
+  /**
+   * Get nested properties by path
+   * @param value Value of an object
+   * @param path Path to the property. If not present, the root value will be
+   * returned.
+   */
+  static getDeepProperty(value: BoundValue, path?: string) {
```

Comment:
https://github.com/strongloop/loopback-next/pull/671#discussion_r160131773

The `Binding` class is already too large, let's not make it a kitchen sink of everything loosely related to retrieving bound values. I have intentionally moved `getDeepProperty` out of a static member into a standalone function. I understand a private helper in `src/context.ts` may not be the right place, since it looks like you need to access this function from more places. In that case please move it to a different file, in the worst case create `src/util.ts` or similar (with the risk of eventually getting the kitchen sink again, but this time in a different file).

Reply:
Fixed in another PR.